src/components/shared/MarkerTooltipContents.js

```diff
@@ -53,52 +86,149 @@ function getMarkerDetails(data: MarkerPayload): React$Element<*> | null {
+                {_markerDetail(
+                  'gcmaxpause',
+                  'Max Pause',
+                  timings.max_pause,
+                  x => x + 'ms'
+                )}
+                {_markerDetail(
+                  'gcusage',
+                  'Heap usage',
+                  _formatBytes(timings.allocated)
+                )}
+                {_markerDetail(
+                  'gcmmu20ms',
+                  'MMU 20ms',
+                  timings.mmu_20ms,
+                  _formatPercent
+                )}
+                {_markerDetail(
+                  'gcmmu50ms',
+                  'MMU 50ms',
+                  timings.mmu_50ms,
+                  _formatPercent
+                )}
+                {_markerDetail('gcnumminors', 'Minor GCs', timings.minor_gcs)}
+                {_markerDetail('gcnumslices', 'Slices', timings.slices)}
+                {_markerDetail('gcnumzones', 'Zones', zones_collected_total)}
+                {_markerDetail(
+                  'gcnumcompartments',
+                  'Compartments',
+                  timings.total_compartments
+                )}
+              </div>
+            );
+          }
+          default:
+            return null;
         }
-        return (
-          <div className="tooltipDetails">
-            {_markerDetail('gcreason', 'Reason', data.timings.reason)}
-            {data.timings.nonincremental_reason !== 'None' &&
-              _markerDetail(
-                'gcnonincrementalreason',
-                'Non-incremental reason',
-                data.timings.nonincremental_reason
-              )}
-            {_markerDetail(
-              'gcmaxpause',
-              'Max Pause',
-              data.timings.max_pause,
-              x => x + 'ms'
-            )}
-            {_markerDetail('gcnumminors', 'Minor GCs', data.timings.minor_gcs)}
-            {_markerDetail('gcnumslices', 'Slices', data.timings.slices)}
-            {_markerDetail('gcnumzones', 'Zones', zones_collected_total)}
-          </div>
-        );
       }
       case 'GCSlice': {
+        const timings = data.timings;
+        let triggers;
+        if (timings.trigger_amount && timings.trigger_threshold) {
+          triggers = _markerDetail(
+            'gctrigger',
+            'Trigger (amt/trig)',
+            timings.trigger_amount + ' / ' + timings.trigger_threshold
+          );
+        }
         return (
           <div className="tooltipDetails">
-            {_markerDetail('gcreason', 'Reason', data.timings.reason)}
-            {_markerDetail('gcbudget', 'Budget', data.timings.budget)}
+            {_markerDetail('gcreason', 'Reason', timings.reason)}
+            {_markerDetail('gcbudget', 'Budget', timings.budget)}
             {_markerDetail(
               'gcstate',
               'States',
-              data.timings.initial_state + ' \u2013 ' + data.timings.final_state
+              timings.initial_state + ' \u2013 ' + timings.final_state
```

Comment:
nit: replace the `\u2013` with the proper unicode character (I believe this is `–`)